tests/integration/test_wpk/test_wpk_agent.py

```diff
@@ -10,24 +10,29 @@
 import subprocess
 import yaml
 import json
+import re
 
+from wazuh_testing import tools
+from wazuh_testing.tools.monitoring import make_callback, FileMonitor
 from configobj import ConfigObj
```

Comment:
This import is no longer being used, please remove it.

Reply:
Done 29484935443c9d15880a267f5abcd72f8779aa3a